### PR TITLE
KAA-932 Client bug fixes

### PR DESCRIPTION
**Pull request by Dubland**

### CODE REVIEW

**Comment by forGGe** [comment]
Looks good for me

File: client/client-multi/client-c/test/kaatcp/kaatcp_request_test.c [hunk 1]
```diff
@@ -140,6 +140,35 @@ void test_kaatcp_ping(void)
...
+	KAA_TRACE_IN(logger);
+	kaatcp_kaasync_t kaa_sync_message;
+	kaa_sync_message.sync_request_size = 3;
+	kaa_sync_message.sync_request = "hi";
+	unsigned char buffer[100] = "";
+	size_t buffer_size = 10;
+	memset(buffer, 0xEA, 100);
+	ASSERT_EQUAL(kaatcp_get_request_kaasync(&kaa_sync_message, buffer, &buffer_size), KAATCP_ERR_BUFFER_NOT_ENOUGH);
+	for (int i = 10; i < 100; i++) {
+		ASSERT_EQUAL(buffer[i], 0xEA);
```
Comment (ohamov) [inline client/client-multi/client-c/test/kaatcp/kaatcp_request_test.c]: indent

**Comment by ohamov** [comment]
LGTM

**Comment by forGGe** [comment]
Merge postponed to further releases.

**Pull request closed without merging**